TransIT.BLL.Tests/Services/CrudServiceTests.cs

```diff
@@ -159,11 +152,18 @@ public async Task DeleteAsync_GivenExistingEntityId_DeletesEntity(int id)
         public async Task DeleteAsync_GivenNonExistingEntityId_ReturnsNothing(int id)
         {
             int previousCount = _context.Count;
+            try
+            {
+                await _crudService.DeleteAsync(id);
 
-            await _crudService.DeleteAsync(id);
+            }
+            catch (Exception)
+            {
+
+            }
```

Comment:
- the same here, try to use framework for this, ex. look in the previous comment